Dockerfile

```diff
@@ -15,11 +15,9 @@ RUN pip install poetry && \
   poetry config virtualenvs.create false && \
   poetry install --no-dev
 
-COPY /src/ /app
-COPY /etc/ /app
-
+COPY /src /app/src
+COPY /etc /app/etc
 
 WORKDIR /app/src
 
-ENTRYPOINT [ "/app/etc/run.sh" ]
```

Comment:
Why use `CMD` and not `ENTRYPOINT`?

Reply:
`CMD` is the correct instruction to run the software contained by the image 

https://docs.docker.com/develop/develop-images/instructions/#cmd

Reply:
But `ENTRYPOINT` is the default command the image should run. I think we can stick with ENTRYPOINT for now.

Reference: https://docs.docker.com/develop/develop-images/instructions/#entrypoint